third_party/airflow/armada/operators/utils.py

```diff
@@ -94,6 +94,8 @@ def search_for_job_complete(
             job_status_return = job_status_callable(
                 queue=queue, job_id=job_id, job_set_id=job_set_id
             )
+
+        time.sleep(3)
```

Comment:
:question: 

If this is needed, it should be commented and an issue created to remove the need for a sleep in the future

Reply:
Is it big deal to add a sleep so we aren’t polling the api every nanosecond?  

It’s prob leftover debugging but it spams the grpc logging bcuz every rpm call gets logged

Reply:
Oh, that's fair. I generally comment any `sleep()` calls in code though because they tend to catch the eye